executor.go

```diff
@@ -594,10 +594,6 @@ func completeValue(eCtx *ExecutionContext, returnType Type, fieldASTs []*ast.Fie
 	// If field type is Scalar or Enum, serialize to a valid value, returning
 	// null if serialization is not possible.
 	if returnType, ok := returnType.(*Scalar); ok {
-		err := invariant(returnType.Serialize != nil, "Missing serialize method on type")
```

Comment:
I was wondering about this code elimination, but it does totally make sense :+1:  ... we would get compile error if we would make`Scalar.Serialize ` or `Enum.Serialize` actually `nil`.